plugins/transfer/plugin.go

```diff
@@ -156,6 +161,9 @@ type transferConfig struct {
 	// MaxConcurrentUploadedLayers is the max concurrent uploads for push
 	MaxConcurrentUploadedLayers int `toml:"max_concurrent_uploaded_layers"`
 
+	// UnpackConfiguration is used only when CheckPlatformSupported is true
+	CheckPlatformSupported bool `toml:"check_platform_supported"`
```

Comment:
nit: comment should start with its own name, suggest to change to something like `CheckPlatformSupported enables platform check specified in UnpackConfiguration`, and move `UnpackConfiguration is used only when ...` to its own comment section.

Also when we should/shouldn't enable this?